src/util/config/EnvConfig.ts

```diff
@@ -0,0 +1,60 @@
+/*
+	Spacebar: A FOSS re-implementation and extension of the Discord.com backend.
+	Copyright (C) 2025 Spacebar and Spacebar Contributors
+	
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published
+	by the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { LogEnvConfiguration } from "./envTypes/LogEnvConfiguration";
+import { DatabaseEnvConfiguration } from "./envTypes/DatabaseEnvConfiguration";
+import os from "os";
+import { ConfigurationEnvConfiguration } from "./envTypes/ConfigurationEnvConfiguration";
+
+const logConfig = new LogEnvConfiguration();
+const databaseConfig = new DatabaseEnvConfiguration();
+const configurationConfig = new ConfigurationEnvConfiguration();
```

Comment:
bit verbose to have 4 additional classes just for env var getters...?

Reply:
Also a requirement for having the deprecation fallbacks, unless you really wanna have them all in a single class? It's now 6 classes fwiw